addon/utils/group-utils.js

```diff
@@ -80,7 +80,7 @@ export function filterOptions(options, text, matcher, skipDisabled = false) {
     if (isGroup(entry)) {
       let suboptions = filterOptions(get(entry, 'options'), text, matcher, skipDisabled);
       if (get(suboptions, 'length') > 0) {
-        opts.push({ groupName: entry.groupName, options: suboptions });
```

Comment:
I'm not sure if this should be smart enough to not add the property `disabled` if the original doesn't have it.
```
let groupCopy = { groupName: entry.groupName, options: suboptions };
if (entry.hasOwnProperty('disabled')) {
  groupCopy.disabled = entry.disabled;
}
```

will make the test pass without changes. Do that.